pkg/autodiscover/autodiscover_operators.go

```diff
@@ -19,11 +19,14 @@ package autodiscover
 import (
 	"context"
 
+	helmclient "github.com/mittwald/go-helm-client"
```

Comment:
Use the openshift helm client-go instead:
	clienthelm "github.com/openshift/client-go/helm/clientset/versioned/typed/helm/v1beta1"

Reply:
Disregard, this only list helm repositories.
deliedit@fedora newrepo]$ oc get helmchartrepositories -owide
NAME                    AGE
openshift-helm-charts   41d
[deliedit@fedora newrepo]$